packages/mobile/src/transactions/GoldTransactionFeedItem.tsx

```diff
@@ -0,0 +1,155 @@
+import Touchable from '@celo/react-components/components/Touchable'
+import colors from '@celo/react-components/styles/colors.v2'
+import fontStyles from '@celo/react-components/styles/fonts.v2'
+import variables from '@celo/react-components/styles/variables'
+import gql from 'graphql-tag'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { ExchangeItemFragment } from 'src/apollo/types'
+import CurrencyDisplay from 'src/components/CurrencyDisplay'
+import { CURRENCIES, CURRENCY_ENUM } from 'src/geth/consts'
+import { Namespaces } from 'src/i18n'
+import { navigateToExchangeReview } from 'src/transactions/actions'
+import { TransactionStatus } from 'src/transactions/reducer'
+import { getDatetimeDisplayString } from 'src/utils/time'
+
+type Props = ExchangeItemFragment & {
+  status?: TransactionStatus
+}
+
+export function ExchangeFeedItem(props: Props) {
+  const { t, i18n } = useTranslation(Namespaces.walletFlow5)
+  const { amount, makerAmount, takerAmount, status, timestamp } = props
+  const onPress = () => {
+    navigateToExchangeReview(timestamp, {
+      makerAmount,
+      takerAmount,
+    })
+  }
+
+  const isSellGoldTx = makerAmount.currencyCode === CURRENCIES[CURRENCY_ENUM.GOLD].code
+  const dateTimeFormatted = getDatetimeDisplayString(timestamp, i18n)
+  const isPending = status === TransactionStatus.Pending
+  // We always show Local Currency to cGLD exchage rate
+  // independent of transaction type
+  const localAmount = (isSellGoldTx ? makerAmount : takerAmount).localAmount!
+  // TODO: find a way on how to show local exchangeRate without this hack
+  const exchangeRateAmount = {
+    value: localAmount.exchangeRate,
+    currencyCode: CURRENCIES[CURRENCY_ENUM.DOLLAR].code,
+    localAmount: {
+      value: localAmount.exchangeRate,
+      exchangeRate: localAmount.exchangeRate,
+      currencyCode: localAmount.currencyCode,
+    },
+  }
+
+  return (
+    <Touchable onPress={onPress}>
+      <View style={styles.container}>
+        <View style={styles.firstRow}>
+          <View style={styles.desc}>
+            <Text style={styles.txMode}>
+              {isSellGoldTx ? t('feedItemGoldSold') : t('feedItemGoldPurchased')}
+            </Text>
+            <>
+              <Text style={styles.exchangeRate}> @ </Text>
+              <CurrencyDisplay
+                amount={exchangeRateAmount}
+                hideSymbol={false}
+                hideCode={true}
+                showLocalAmount={true}
+                style={styles.exchangeRate}
+              />
+            </>
```

Comment:
If you're only interested in displaying a specific value and not doing conversions, it looks like you can simplify to this:

```jsx
const localAmount = (isSellGoldTx ? makerAmount : takerAmount).localAmount!
const exchangeRateAmount = {
  value: localAmount.exchangeRate,
  currencyCode: localAmount.currencyCode,
  localAmount: null
}

// Then
<CurrencyDisplay
  amount={exchangeRateAmount}
  hideSymbol={false}
  hideCode={true}
  showLocalAmount={false}
  style={styles.exchangeRate}
/>
```

Reply:
Unfortunately this would work with Symbols. See here:
https://github.com/celo-org/celo-monorepo/blob/337d28c94e418afea2d6ec05f5a7930916fc8371/packages/mobile/src/components/CurrencyDisplay.tsx#L151-L155

Reply:
I think should decouple conversion and representation inside `CurrencyDisplay`, but that is for different PR :)

Reply:
ah 😢, I intended to merge LocalCurrency and CURRENCIES to avoid this kind of mess.
All good then 👍